cityofaustin/atd-vz-data
# Ability to view temp records in the crashes list and also soft-delete them

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/18429

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Start up your VZ data model, run the migrations and apply metadata from this branch. You can run the cris import script to get some more data.
2. Create some temp records with varying units, injuries, etc.
3. Test deleting the temp record. In the database make sure the crash and all its associated units and people records all had their `is_deleted` columns updated to `true`.
4. Make sure the temp records you deleted are no longer showing up in the crashes list page, the fatalities page, or in the dashboard totals.
5. Will crash if you try to go to the old temp crash details page but I think there is another issue open to address routing to the 404 page so i think we can address this in that PR


---
#### Ship list
